setup.cfg

```diff
@@ -1,47 +0,0 @@
-[metadata]
-name = velocitas_model_generator
-version = 0.4.0
```

Comment:
how/where is this version now specified?

Reply:
Version is now dynamically read from git tag